src/meta/model_v2/migration/src/m20230908_072257_init.rs

```diff
@@ -723,6 +723,7 @@ impl MigrationTrait for Migration {
                     .col(ColumnDef::new(Function::Link).string())
                     .col(ColumnDef::new(Function::Identifier).string())
                     .col(ColumnDef::new(Function::Body).string())
+                    .col(ColumnDef::new(Function::CompressedBinary).string())
```

Comment:
Well, suggest to cherry pick this PR to release-1.7 after it get merged. Because after the release, we need to guarantee backward compatibility, table schema changes and data correctness between each version should be managed in different migration file.

Reply:
Oh, I found that there are already several PRs that includes table schema changes but not cherry-picked to release branch. Since the single node requires the unready feature `sql-backend`, just talked to @kwannoel , we gonna create a new tag from main to release single node.